curriculum/challenges/english/10-coding-interview-prep/rosetta-code/day-of-the-week.english.md

```diff
@@ -23,11 +23,11 @@ Write a function that takes a start year and an end year and return an array of
 tests:
   - text: <code>findXmasSunday</code> should be a function.
     testString: assert(typeof findXmasSunday === 'function');
-  - text: <code>findChristmasSunday(2000, 2100)</code> should return an array.
+  - text: <code>findXmasSunday(2000, 2100)</code> should return an array.
     testString: assert(typeof findXmasSunday(2000, 2100) === 'object');
-  - text: <code>findChristmasSunday(2008, 2121</code> should return [1977, 1983, 1988, 1994, 2005, 2011, 2016]
+  - text: <code>findXmasSunday(2008, 2121)</code> should return [1977, 1983, 1988, 1994, 2005, 2011, 2016]
     testString: assert.deepEqual(findXmasSunday(1970, 2017), firstSolution);
-  - text: <code>findChristmasSunday(2008, 2121</code> should return [2011, 2016, 2022, 2033, 2039, 2044, 2050, 2061, 2067, 2072, 2078, 2089, 2095, 2101, 2107, 2112, 2118]
+  - text: <code>findXmasSunday(2008, 2121</code> should return [2011, 2016, 2022, 2033, 2039, 2044, 2050, 2061, 2067, 2072, 2078, 2089, 2095, 2101, 2107, 2112, 2118]
```

Comment:
```suggestion
  - text: <code>findXmasSunday(2008, 2121)</code> should return [2011, 2016, 2022, 2033, 2039, 2044, 2050, 2061, 2067, 2072, 2078, 2089, 2095, 2101, 2107, 2112, 2118]
```

Reply:
@Sky020 I did not catch that.  Thanks.

>Also, @RandellDawson , should the expected return not also be within <code> tags?

Yes, I did not notice those either.

Reply:
I added the <code> tags to the two returns and squashed the commits for a cleaner commit history. 
let me know if you see/want anything else!